CHANGELOG.md

```diff
@@ -64,6 +64,14 @@ Added the ability to configure the WASM module used for assets canisters through
 
 ### chore: Add a message that `redeem_faucet_coupon` may take a while to complete
 
+### feat: dfx deploy <frontend canister name> --by-proposal
+
+This supports asset updates through SNS proposal.
+
+Uploads asset changes to an asset canister (propose_commit_batch()), but does not commit them.
+
+The SNS will call `commit_proposed_batch()` to commit the changes.  If the proposal fails, the caller of `dfx deploy --by-proposal` should call `delete_batch()`.
```

Comment:
can SNS also call `delete_batch()`?

Reply:
Yes, delete_batch requires `Prepare` permission, and `Commit` permission can also do anything `Prepare` can do.  From my perspective it would be nice if some day SNS would either call `commit_proposed_batch` (for proposal approved) or `delete_batch` (for proposal rejected), but I do not know if that is where they are going.